flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvRowDataSerDeSchemaTest.java

```diff
@@ -109,6 +109,54 @@ public void testSerializeDeserialize() throws Exception {
 			new byte[] {107, 3, 11});
 	}
 
+	@Test
+	public void testSerializeDeserializeForTime() throws Exception {
+		testFieldForTime(
+			TIME(3),
+			"12:12:12.232",
+			"12:12:12.232",
+			(deserSchema) -> deserSchema.setNullLiteral("null"),
+			(serSchema) -> serSchema.setNullLiteral("null"),
+			",");
+		testFieldForTime(
+			TIME(3),
+			"12:12:12.232421",
+			"12:12:12.232",
+			(deserSchema) -> deserSchema.setNullLiteral("null"),
+			(serSchema) -> serSchema.setNullLiteral("null"),
+			",");
+		testFieldForTime(
+			TIME(3),
+			"12:12:12.23",
+			"12:12:12.23",
+			(deserSchema) -> deserSchema.setNullLiteral("null"),
+			(serSchema) -> serSchema.setNullLiteral("null"),
+			",");
+		testFieldForTime(
+			TIME(0),
```

Comment:
Add more tests for `TIME(1)`, `TIME(2)`.

Reply:
ok